deprecation-index.yml

```diff
@@ -54,7 +54,7 @@
     - FilePrepareDirectoryStatic.php
 'file_scan_directory()':
   Rector: FileScanDirectoryRector.php
-  PHPStan: 'Call to deprecated function file_scan_directory(). Deprecated in drupal:8.8.0 and is removed from drupal:9.0.0. Use Drupal​\​Core​\​File​\​FileSystemInterface::scanDirectory() instead.'
+  PHPStan: 'Call to deprecated function file_scan_directory(). Deprecated in drupal:8.8.0 and is removed from drupal:9.0.0. Use Drupal\Core\File\FileSystemInterface::scanDirectory() instead.'
```

Comment:
You can see there is difference in the slash in the line but there is no visual difference. This appears to be the character https://www.fileformat.info/info/unicode/char/200b/index.htm

The only way I could see it was editing the file in vim🤷‍♂️

Reply:
I searched and I didn't see anymore of these characters in the project.